gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -29,6 +29,9 @@
 from gnomad.utils.constraint import build_models
 from gnomad.utils.filtering import filter_x_nonpar, filter_y_nonpar
 from gnomad.utils.reference_genome import get_reference_genome
+from gnomad_qc.resource_utils import (
+    check_resource_existence,
+)  # TODO: fix throughout code
```

Comment:
Is this still a TODO, or is it all fixed?

Reply:
all fixed